analyzer/validatorstakinghistory/validatorstakinghistory.go

```diff
@@ -148,6 +148,17 @@ func (p *processor) ProcessItem(ctx context.Context, batch *storage.QueryBatch,
 			validators[vID] = struct{}{}
 		}
 	}
+	// Download staking events for the first block, which includes validator staking rewards.
+	stakingEvents, err := p.source.StakingEvents(ctx, epoch.startHeight)
+	if err != nil {
+		return fmt.Errorf("downloading staking events for height %d", epoch.startHeight)
```

Comment:
woa a lot of places in this file discard err

Reply:
good eye, fixed throughout